package.json

```diff
@@ -34,6 +34,8 @@
   },
   "homepage": "https://github.com/RAIRLab/PeirceMyHeart#readme",
   "devDependencies": {
+    "@types/file-saver": "^2.0.7",
+    "@types/node": "20.12.7",
     "@types/node": "20.12.8",
```

Comment:
I am very worried about two conflicting types for node, here and in package-lock.json. Otherwise everything tests perfectly.

Reply:
Good catch, this was the result of a failed merge.